avm/ptn/authorization/resource-role-assignment/main.bicep

```
@@ -0,0 +1,108 @@
+metadata name = 'ResourceRole Assignments (All scopes)'
```

```suggestion
metadata name = 'Resource-scoped role assignment'
```

Just thinking this renaming would make sense as 'all scopes' is technically not possible (as you cannot deploy this module to e.g. a management group scope). Just a minor detail.

After updating please run the `Set-AVMModule` script again.

---

That's my last comment. Everything else looks good from my end. The most one could argue is that you presumably don't need the `!` at the `name` output anymore. But that's a nitpick so ... yeah 😄